src/providers/twitch/api/README.md

```diff
@@ -178,47 +178,47 @@ Used in:
 
 - `controllers/commands/CommandController.cpp` to send Twitch native shoutout using "/shoutout <username>"
 
+### Warn Chat User
+
+URL: https://dev.twitch.tv/docs/api/reference/#warn-chat-user
+
+Used in:
+
+- `controllers/commands/CommandController.cpp` to warn users via "/warn" command
```

Comment:
I don't think we should mention the uses here - they're bound to be out of sync, and it's not hard to find references to the functions.

Reply:
yeah, it was useful during the big helix rework but it's fine to not update this

since you've already updated it, it's fine to leave it, but the file will probably be removed/refactored at some point